### PR TITLE
Watchlist CLI

### PR DESCRIPTION
**Pull request by ajvb**
Add a CLI to Watchlist that allows of creation of watchlist entry json
blobs with support for submitting them to datastore.

Additionally, include documentation for using the Minfraud CLI.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,6 +40,59 @@ using these identifiers when the pipeline is executing.
...
 during pipeline execution.
 
+### Interacting with Minfraud
+
+Reputation data can be fetched from Minfraud locally using the main method in the [`Minfraud`](https://mozilla-services.github.io/foxsec-pipeline/secops-beam/com/mozilla/secops/Minfraud.html) class.
+
+You must provide the accountid and licensekey, plus the IP and/or email you want to get reputation data for:
+
+```bash
+bin/m exec:java -Dexec.mainClass="com.mozilla.secops.Minfraud" -Dexec.args="-a 'cloudkms://...' -l 'cloudkms://...' -i '8.8.8.8' -e 'example@example.com'"
```
Comment (ameihm0912) [inline README.md]: Is there a way to clarify the arguments here? It's unclear what is going on unless you look at the code.

Comment (ajvb) [reply README.md]: Yep! Fixed.

**Comment by ajvb** [comment]
@ameihm0912 Sure? I'm happy to move it too it's own document but am unclear on the value at this point. The README is still really short with this addition imo.

**Comment by ameihm0912** [comment]
> @ameihm0912 Sure? I'm happy to move it to it's own document but am unclear on the value at this point. The README is still really short with this addition imo.

Yeah not critical right now, just thought it looked strange to have most of the top-level README CLI documentation.

**Comment by ajvb** [comment]
@ameihm0912 Ah I see what you mean, yeah I agree. Let me at least re-arrange it for now.

**Pull request merged**